src/Management/src/EndpointBase/Metrics/ServiceCollectionExtensions.cs

```diff
@@ -105,7 +105,7 @@ public static IServiceCollection AddOpenTelemetryMetricsForSteeltoe(this IServic
                         .AddRegisteredViews(views)
                         .AddExporters(exporters);
 
-                    var wavefrontExporter = provider.GetService<WavefrontMetricsExporter>();
+                    var wavefrontExporter = provider.GetService<WavefrontMetricsExporter>(); // Not an IMetricsExporter
```

Comment:
The code as it reads, collects all Imetrics exporters but since this one isn't, we have ask for it by a Type (rather than interface). Reading back through the code, I was confused so I thought the comment might help .